requirements.txt

```diff
@@ -312,7 +312,7 @@ pymacaroons==0.13.0
     #   surl
 pymdown-extensions==8.1.1
     # via mkdocs-material
-pynacl==1.1.2
+pynacl==1.2.0
```

Comment:
Bumped due to this issue https://github.com/freedomofpress/securedrop/issues/6023